### ISSUE TITLE
ff.create_gantt uses deprecated pandas' methods and return warnings

### ISSUE THREAD

**Issue by ricgu8086** [plotly/plotly.py#2066]
When executing the following code

```
import plotly.figure_factory as ff

fig2 = ff.create_gantt(df_total_gantt, colors=['#FF0000', '#0000FF'], index_col='Complete',
                      show_colorbar=True, bar_width=0.2, showgrid_x=True, showgrid_y=True)
fig2.show()
```

it is executed correctly and returns a nice plot, however, it also returns the following:

>  /usr/local/lib/python3.6/dist-packages/plotly/figure_factory/_gantt.py:49: FutureWarning:
> 
> 
> .ix is deprecated. Please use
> .loc for label based indexing or
> .iloc for positional indexing
> 
> See the documentation here:
> http://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#ix-indexer-is-deprecated

**Comment by emmanuelle** [comment]
Thank you @ricgu8086 ! We'll fix this, unless of course you would like to submit a pull request here :-).

**Comment by nicolaskruchten** [comment]
This was already patched actually and will come out in 4.5 :) See https://github.com/plotly/plotly.py/pull/2011
